src/services/directDonationsReport/directDonationsReport.service.js

```diff
@@ -0,0 +1,76 @@
+const config = require('config');
+const { Parser } = require('json2csv');
+const { getTokenByAddress } = require('../../utils/tokenHelper');
+
+const createAggregateQuery = ownerType => {
+  return [
+    { $match: { homeTxHash: { $exists: true }, ownerType } },
+    {
+      $lookup: {
+        from: `${ownerType}s`,
+        let: { id: { $toObjectId: '$ownerTypeId' } },
+        pipeline: [
+          {
+            $match: { $expr: { $eq: ['$_id', '$$id'] } },
+          },
+        ],
+        as: ownerType,
+      },
+    },
+    {
+      $unwind: `$${ownerType}`,
+    },
+  ];
+};
+const normalizeDonation = donation => {
+  const dappUrl = config.get('dappUrl');
+  const { ownerType, campaign, milestone, tokenAddress } = donation;
+  const token = getTokenByAddress(tokenAddress);
+  const data = {
+    _id: donation._id,
+    usdValue: donation.usdValue,
+    amount: Number(donation.amount).toExponential(),
+    amountHumanized: donation.amount / 10 ** 18,
```

Comment:
rename it to amount and use weitoEth function

Reply:
done